internal/envoy/sds_test.go

```diff
@@ -259,12 +262,13 @@ func runTestServer(t *testing.T, ca []byte, registryFn func(*gomock.Controller)
 		Name: "test",
 	}, time.Now(), nil)
 
-	sds := NewSDSServer(hclog.NewNullLogger(), fetcher, secretClient, memory.NewStore(memory.StoreConfig{}))
-	sds.bindAddress = serverAddress
-	sds.protocol = "unix"
+	var store store.Store = storeMocks.NewMockStore(ctrl)
 	if registryFn != nil {
-		sds.store = registryFn(ctrl)
+		store = registryFn(ctrl)
```

Comment:
Some tests will provide a func to build the store themselves. If they do, use it instead of the mock store